afids_utils/tests/test_ext.py

```diff
@@ -164,32 +164,29 @@ def test_valid_get_afids(self, valid_fcsv_file: PathLike[str], label: int):
 
 
 class TestSaveFcsv:
-    @given(afids_coords=afid_coords())
+    @given(afids_set=afid_set())
```

Comment:
Nitpick: I'd rather call this think `afid_set` for consistency with the class name